right_arm:
    power_ip: 10.6.10.105
    data_ip: 192.168.1.1
